main.py

```diff
@@ -784,6 +784,7 @@ def stream_content():
 
 class PerformFiltersRequest(BaseModel):
     enabled_filters: List[str]
+    ban_list: Optional[List[str]] = []
```

Comment:
can we make this a config parameter(hash), and let ban_list pass through it. I believe we might have a lot of config params getting to the input args here.